### DISCUSSION TITLE
How to add plugins to rich text editor?

### DISCUSSION THREAD

**Discussion by KeeganFargher** [mantinedev/mantine#742] [Q&A] [answered]
Hi

I want to add this module [https://github.com/reedsy/quill-cursors#configuration](https://github.com/reedsy/quill-cursors#configuration) to the [rich text library](https://mantine.dev/others/rte/) but I have no idea how to access the "modules" prop? 

Thanks!

**Comment by rtivital** [comment, marked as answer]
Currently, that's not supported, I'll add option to provide modules as a prop in next patch (3.6.4)

**Reply by KeeganFargher** [reply]
Thank you 👻

**Comment by FlyingDragon7777** [comment]
Whats up guys. Im new here and i have a little question. I would like to enter the resale business and am looking for a good and fast bot that tries to get the limited shoes on several accounts at the same time
